CODEOWNERS

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-# FireFly Core Maintainers
-* @peterbroadhurst @nguyer @awrichar
+# FireFly transaction manager maintainers
```

Comment:
Not sure the comment is necessary anymore given that the team name is self-describing